src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -154,26 +154,42 @@ export const BrowserTrackConfig = (props: BrowserTrackConfigProps) => {
           </button>
         </dd>
         <dd className={styles.heightSwitcher}>
-          <button className={styles.trackHeightBtn}>
-            <img src={trackHeightBtn} />
-          </button>
+          <div className={styles.trackHeightBtn}>
+            <ImageButton
+              buttonStatus={Status.INACTIVE}
+              image={browserTrackConfig.trackHeightBtn.icon}
+              description={browserTrackConfig.trackHeightBtn.description}
+            />
+          </div>
         </dd>
       </dl>
       <dl>
```

Comment:
Now that we are slowly phasing out `dl` in favour of `ul`, maybe it is time to change this to `ul`. What do you think @azangru?

Reply:
I don't think it even needs the semantic of a list. And a `ul` implies `li`s as its children. I vote for simple trusty `div`s instead of all those lists.

Reply:
Decided to address this in a separate PR:

![image](https://user-images.githubusercontent.com/6834224/73661160-842d8780-4691-11ea-97c6-4c26ea4f4ca9.png)